platform/swift/source/src/conversion.rs

```diff
@@ -0,0 +1,446 @@
+// capture-sdk - bitdrift's client SDK
+// Copyright Bitdrift, Inc. All rights reserved.
+//
+// Use of this source code is governed by a source available license that can be found in the
+// LICENSE file or at:
+// https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt
+
+// FFI test functions for Swift/Objective-C integration tests
+// These need to be available in all builds since Swift tests call them via FFI
+#[path = "./conversion_tests.rs"]
```

Comment:
We need to solve this somehow. There has to be a way to include `conversion_tests.rs` in the integration test build without them bleeding into the release build!